### PR TITLE
[CF-458] Set enable_dhcp to original value on destination.

### PR DESCRIPTION
**Pull request by antonf**
The bug was introduced which caused all subnets change its enable_dhcp to
False during VM migration. This patch fixes this.

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

File: cloudferry/lib/os/network/network_utils.py [hunk 1]
```diff
@@ -81,17 +76,20 @@ def get_tenants(instances):
...
                 network_resource.reset_subnet_dhcp(snet['id'], False)
-            else:
-                # Reset DHCP to the original settings
-                network_resource.reset_subnet_dhcp(snet['id'],
-                                                   snet['enable_dhcp'])
+                disabled_subnets.append(snet['id'])
+        yield
+    finally:
+        for subnet_id in reversed(disabled_subnets):
+            network_resource.reset_subnet_dhcp(subnet_id, True)
```
Comment (roman-verchikov) [inline cloudferry/lib/os/network/network_utils.py]: I guess we should reset to original value for dhcp, not `True`

**Comment by plomakin** [comment]
+1
tested with #994 and it works:
Validate subnets were migrated with correct parameters. ... ok

**Comment by roman-verchikov** [comment]
+1, can be merged

**Pull request merged**